modyn/selector/internal/selector_strategies/new_data_strategy.py

```diff
@@ -86,6 +88,16 @@ def _get_data_reset(self) -> Iterable[list[int]]:
 
             yield samples
 
+    def _get_data_tail(self) -> Iterable[list[int]]:
+        assert not self.reset_after_trigger and self.tail_triggers > 0
+
+        for samples in self._get_tail_triggers_data():
```

Comment:
Can you please add tests for the NewDataStrategy testing for correct behavior :)? Thanks!